ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,100 @@
+265.1 "--untargetable--"
+274.4 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+276.6 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+282.6 "--targetable--"
+285.7 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+295.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+313.1 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+332.3 "Elegeia Unforgotten" sync / 1[56]:[^:]*:The Endsinger:6682:/
+357.5 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+369.7 "Galaxias" sync / 1[56]:[^:]*:The Endsinger:6C69:/
+375.9 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+388.0 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+401.2 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+407.2 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+417.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+426.4 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+436.6 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+446.2 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+450.8 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+464.0 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+483.2 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+490.3 "--untargetable--"
+499.6 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+501.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+507.8 "--targetable--"
+513.9 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+519.9 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+539.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+549.4 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+559.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+563.5 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+576.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+
+# Phase 2, add spawns
+1000.0 "Planetes" sync / 1[56]:[^:]*:The Endsinger:6B58:/ window 1000,0
+1000.0 "--untargetable--"
+1014.3 "--targetable--"
+1023.4 "Meteor Radiant" sync / 1[56]:[^:]*:Kakodaimon:6656:/
+1035.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1040.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1045.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1050.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1056.6 "Meteor Outburst" sync / 1[56]:[^:]*:Kakodaimon:665C:/
+# Loop Katastrophe raidwide until enrage
+1067.8 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
+1078.9 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/ jump 1067.8
+1090.0 "Katastrophe"
+1101.1 "Katastrophe"
+1112.2 "Katastrophe"
+1123.3 "Katastrophe"
+1200.0 "--sync--" sync / 00:0039::10 seconds until oblivion is sealed/ window 200,0
+1212.2 "Ultimate Fate" sync / 1[56]:[^:]*:The Endsinger:6B59:/ window 10,10
+1218.9 "--targetable--"
+
+# Phase 3, loops until soft enrage
+2000.0 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6B5B:/ window 1000,0
+2036.0 "--untargetable--"
+2074.0 "--targetable--"
+2086.1 "Telos" sync / 1[56]:[^:]*:The Endsinger:665E:/
+2099.3 "Telomania (cast)" sync / 1[56]:[^:]*:The Endsinger:665F:/ window 0,30
```

Comment:
```suggestion
2099.3 "Telomania (cast)" sync / 1[56]:[^:]*:The Endsinger:665F:/
```

I think no need for a window here, as well, since you have the jump below.